src/Domain/Apprenticeship/ApprenticeshipEpisode.cs

```diff
@@ -162,8 +162,11 @@ public void UpdateMathsAndEnglishCourses(List<MathsAndEnglish> mathsAndEnglishCo
     /// </summary>
     public void UpdateCompletion(Apprenticeship apprenticeship, DateTime? completionDate, ISystemClockService systemClock)
     {
-        if(!completionDate.HasValue && !_model.CompletionDate.HasValue)
-            return; // No change
+        if ((!completionDate.HasValue && !_model.CompletionDate.HasValue) ||
+            (completionDate.HasValue && _model.CompletionDate.HasValue && completionDate.Value == _model.CompletionDate.Value))
```

Comment:
can we not simplify this with completionDate.GetValueOrDefault() == _model.CompletionDate.GetValueOrDefault()